### ISSUE TITLE
System test fails in Python 3 env

### ISSUE THREAD

**Issue by chengzhuzhang** [E3SM-Project/e3sm_diags#208]
conda create -n e3sm_diags_env_for_phil -c cdat/label/v81 -c conda-forge cdat matplotlib cartopy beautifulsoup4 lxml (system test failed, with python 3)
```
Traceback (most recent call last):
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/bin/e3sm_diags", line 11, in <module>
    load_entry_point('e3sm-diags==1.6.0', 'console_scripts', 'e3sm_diags')()
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/lib/python3.6/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2793, in load_entry_point
    return ep.load()
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2411, in load
    return self.resolve()
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2417, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/lib/python3.6/site-packages/e3sm_diags-1.6.0-py3.6.egg/acme_diags/acme_diags_driver.py", line 24, in <module>
    from acme_diags.acme_viewer import create_viewer
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/lib/python3.6/site-packages/e3sm_diags-1.6.0-py3.6.egg/acme_diags/acme_viewer.py", line 16, in <module>
    from acme_diags.driver.utils.general import get_set_name
  File "/Users/zhang40/anaconda2/envs/e3sm_diags_env_for_phil/lib/python3.6/site-packages/e3sm_diags-1.6.0-py3.6.egg/acme_diags/driver/utils/__init__.py", line 1, in <module>
    import general
ModuleNotFoundError: No module named 'general'
```

conda create -n e3sm_diags_env_for_phil_py2 -c cdat/label/v81 -c conda-forge cdat matplotlib cartopy beautifulsoup4 lxml python=2 (worked)

**Comment by zshaheen** [comment]
@chengzhuzhang I fixed the above issues, but we seem to have some problems with `cdms2` removing the `units` attribute.

**Comment by zshaheen** [comment]
This was fixed in the [v1.7.0](https://github.com/E3SM-Project/e3sm_diags/releases/tag/v1.7.0) release which had Python 3 support.

If I recall correctly, the Python 3 version of ESMF has an issue where after regridding, the `units` attribute is lost. We had a manual fix for this in the `e3sm_diags` code.
